rules/falco_rules.yaml

```diff
@@ -2210,6 +2210,26 @@
   priority: NOTICE
   tags: [network, aws, container, mitre_discovery]
 
+
+# This rule is not enabled by default, since this rule is for cloud environment(GCP, AWS and Azure) only.
+# If you want to enable this rule, overwrite the first macro,
+# And you can filter the container that you want to allow access to metadata by overwriting the second macro.
+- macro: consider_metadata_access
+  condition: (never_true)
+
+- macro: user_known_metadata_access
+  condition: (k8s.ns.name = "kube-system")
+
+# On GCP, AWS and Azure, 169.254.169.254 is a special IP used to fetch
+# metadata about the instance. The metadata could be used to get credentials by attackers.
+- rule: Contact Cloud Instance Metadata Service From Container
```

Comment:
One minor comment, please rename to rule to `Contact cloud metadata service from container`

Reply:
we have a loose naming convention, but this is something we try to follow.

Reply:
@Kaizhe I renamed the name as you suggested.
Could you review it again?
Thank you!